spec/integration/test.js

```diff
@@ -281,6 +287,16 @@ describer('notification api with a live service', () => {
       });
     });
 
+    it('get all received texts', (done) => {
```

Comment:
this `done` pattern sucks if the promise doesn't resolve. If you don't pass in a `done` param, then mocha expects that your test file returns a promise (which it already does), which it just plays out. 

ie: remove `done` from here and the `done()` function call from your `then` callback